pyEpiabm/pyEpiabm/routine/simulation.py

```diff
@@ -51,6 +53,14 @@ def configure(self,
             * `age_stratified`: Boolean to determine whether the output will \
                 be age stratified
 
+        ih_file_params Contains:
+            * `output_dir`: String for the location for the output files, \
+               as a relative path
+            * `status_output`: Boolean to determine whether we need \
+               a csv file containing infection status values
+            * `infectiousness_output`: Boolean to determine whether we need \
+               a csv file containing infectiousness (viral load) values
```

Comment:
What happens if these are both false? I don't think we handle this, but also it doesn't seem obvious from the docs that one should be true. Even just an interpretable error message might be helpful here

Reply:
If both are false then the csv files are not created as the data is never written

Reply:
Oh okay, I wasn't sure if an empty csv was written in that scenario. I think could be good to flag this somewhere just incase, just as a simple warning message?